backend/src/packages/users/user.entity.ts

```diff
@@ -0,0 +1,88 @@
+import { type Entity } from '#libs/interfaces/interfaces.js';
+
+class UserEntity implements Entity {
+  private 'id': number | null;
```

Comment:
let's use # directive for private fields of js

Reply:
I think we should not mix # with typescript access modifiers, and either use javascript approach or typescript

Reply:
So which one?)

Reply:
typescript because it has other modifiers, they are more obvious and we can lint them. But if most of us prefer js, I don't mind

Reply:
Can be resolved, we decided to leave typescript approach